week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -10,17 +10,25 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
+  constructor() {
```

Comment:
Good work here! 
Extra note:
JS classes came in ES6, in ES5 there are constructor functions. 
TypeScript came to cover this issue, so you can use classes but it compile in the background to any version of JS